Userland/Libraries/LibGfx/Bitmap.h

```diff
@@ -1,12 +1,15 @@
 /*
  * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2022, Timothy Slater <tslater2006@gmail.com>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #pragma once
 
 #include <AK/Forward.h>
+#include <AK/Function.h>
+#include <AK/Queue.h>
```

Comment:
Nit: The `<Queue.h>` include can be moved to `Bitmap.cpp`

Reply:
Happy to make this change, but I would like to understand more about why this one can go in the .cpp instead of the .h, I guess I don't know enough about C++ (yet) to understand.

Reply:
The main reason is to avoid unnecessary rebuilds. 

Nothing in `Bitmap.h` needs from `Queue.h`, nor will most places that include `Bitmap.h`. But if you include `Queue.h` in `Bitmap.h`, then make a change to `Queue.h`, everything that includes `Bitmap.h` will also need to be rebuilt.

Reply:
Oh, if you're asking _why_ this can go in the `.cpp` file instead of the header, that's because you don't use the `Queue` type anywhere in the header. You'd only have to include it in the `.h` file if you had a function that had a `Queue` for a parameter or a member that was a `Queue` (for example).

Reply:
Thanks for the explanation (both of them). Makes sense now.